pages/linux/foreman.md

```diff
@@ -0,0 +1,15 @@
+# foreman
+
+> Manage Procfile-based application.
+
+- Start an application:
+
+`foreman start {{process}}`
```

Comment:
I think you should start with the simplest invocation - `foreman start`. And then move to this example.

Also, It will look for the Procfile in the current directory right ?

Reply:
Yup, that's right. Updating now!